rasa/telemetry.py

```diff
@@ -1,5 +1,4 @@
 import asyncio
```

Comment:
`track` is called by all the other `track_` functions. As `track` also uses the `@ensure_telemetry_enabled` decorator, `ensure_telemetry_enabled` will be executed twice for every tracking call. How about making `track_` protected and removing the decorator from it?

Reply:
yes will do